easybuild/easyconfigs/b/bzip2/bzip2-1.0.6.eb

```diff
@@ -12,8 +12,9 @@ toolchainopts = {'pic': True}
 sources = [SOURCE_TAR_GZ]
 source_urls = ['http://www.bzip.org/%(version)s/']
 
-import os
-os.environ['CC'] = 'gcc'
-os.environ['CFLAGS'] = '-O3 -fPIC'
+buildopts = "CC=gcc CFLAGS='-Wall -Winline -O3 -fPIC -g $(BIGFILES)'"
+
+# disable building of shared libraries, doesn't work everywhere (e.g. on OS X, where 'gcc' is actually Clang...)
```

Comment:
well, it was disabled already (without the enhanced of the bzip2 easyblock)

I prefer being able to install bzip2 without shared libs over not installing it, especially since the shared libraries are rarely needed anyway (you can tell by i) the way they need to be installed, ii) a `libbz2.so` isn't even installed by default)